int32[] localization_ids
int32[] motion_ids
---
int32 result
---
int32 feedback